File: src/main/java/net/atos/zac/app/zaken/model/RESTZaakKenmerk.java
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2023-2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.zac.app.zaken.model;

public class RESTZaakKenmerk {

    public String kenmerk;

    public String bron;

    public RESTZaakKenmerk(final String kenmerk, final String bron) {
        this.kenmerk = kenmerk;
        this.bron = bron;
    }
}

File: src/main/kotlin/nl/info/zac/app/zaak/model/RESTZaakKenmerk.kt
/*
 * SPDX-FileCopyrightText: 2021 Atos, 2024 INFO.nl
 * SPDX-License-Identifier: EUPL-1.2+
 */
package nl.info.zac.app.zaak.model

import nl.info.zac.util.AllOpen
import nl.info.zac.util.NoArgConstructor

@NoArgConstructor
@AllOpen
data class RESTZaakKenmerk(
    var kenmerk: String,
    var bron: String
)
